src/commands/config/create.ts

```diff
@@ -0,0 +1,180 @@
+import fs from "fs";
+import { error, info, success } from "signale";
+import { Argv } from "yargs";
+import { deployDocumentStore } from "../../implementations/deploy/document-store";
+import { deployTokenRegistry } from "../../implementations/deploy/token-registry";
+import { readFile } from "../../implementations/utils/disk";
+import { create as CreatedWalletPath } from "../../implementations/wallet/create";
```

Comment:
for the 3rd time, please fix the case!

Reply:
also the name is weird, it doesn't create a path, but a wallet

Reply:
ok fixed.